name_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
```

Comment:
- [ ] Kindly make sure that `nameable` is accessible, you can do this by adding `attr_accessor` ex: `attr_accessor :nameable`